alphagov/paas-cf
# [#141705365] Add MySQL plans

## What

We're slowly enabling the MySQL support in our RDS broker. In preparation, we need to add plans, tests and company of changes to our security groups.

## How to review

- Somewhat dependant on alphagov/paas-rds-broker#50
- Code review
- See travis happy [...]
- Deploy from this branch
- Experience the tests go green
- Run `cf marketplace -s mysql` and see variety of plans
- **Optionally:** Create an instance for your self - _It's covered by the tests_
- Check the plan descriptions and pricing
